src/query.ts

```diff
@@ -12,7 +17,7 @@ export function parseQuery (paramsStr: string = ''): QueryObject {
     const s = (param.match(/([^=]+)=?(.*)/) || [])
     if (s.length < 2) { continue }
     const key = decode(s[1])
```

Comment:
What if key is having space? The space is replaced with + sign while encoding but on decode it doesn't replace back to space.

Reply:
Do you mind to make a PR adding test case?

Reply:
@pi0 Raised PR [https://github.com/unjs/ufo/pull/150]